src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset name="Personal information" class="fieldset">
+        <legend class="legends">
+          Personal information
+        </legend>
+        <label for="surname" class="field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+          >
+        </label>
+        <label for="first_name" class="field">
+          Name:
+          <input type="text" name="first_name">
```

Comment:
better to use camelCase for the name attribute. The point is that you'll may need to interact with it using Javascript, so it will be much easier if its `camelCased`